### PR TITLE
add some template emails

### PR DESCRIPTION
**Pull request by Aariq**
Adds some example email text for communicating with applicants

### CODE REVIEW

File: emails.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+>
+> Dear applicant,
+>
+> Thank you for applying to our Fall Reproducibility and Data Science in R workshop series.
+> After careful review of all applicants, unfortunately your application has not been selected for participation this year.
+> We received many applications and unfortunately we were not able to accept everyone—a difficult decision for us.
+> We encourage you to apply again next year (and remind us that you applied previously when you do) and to continue to reach out to us with data science and reproducibility questions via email, our weekly [drop-in hours](https://datascience.cct.arizona.edu/drop-in-hours), and in the UA Data Science [Slack group](https://jcoliver.github.io/uadatascience-slack/).
+>
+> If you’re up for doing some self-study in lieu of attending the workshop series this Fall, here are some suggested places to get started:
+>
```
Comment (KristinaRiemer) [inline emails.md:27]:
```suggestion
> 
> - Fall workshop series content via the [website for the series](https://cct-datascience.github.io/repro-data-sci/)
> 
```

File: emails.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+> out with any questions you might have.
+
+## Not accepted
+
+> Subject: "Reproducibility and Data Science in R Workshop Series"
+>
+> Dear applicant,
+>
+> Thank you for applying to our Fall Reproducibility and Data Science in R workshop series.
+> After careful review of all applicants, unfortunately your application has not been selected for participation this year.
```
Comment (diazrenata) [inline emails.md]:
```suggestion
> We received a large number of applications this year, and, unfortunately, your application was not selected for participation in this year's workshop. 
```

File: emails.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+> We received many applications and unfortunately we were not able to accept everyone—a difficult decision for us.
```
Comment (diazrenata) [inline emails.md]:
```suggestion
> 
```

**Review by diazrenata — APPROVED** [review]
Just some changes to the language around applications we weren't able to accept.

File: emails.md (hunk 1 above)
Comment (Aariq) [reply emails.md:27]: @diazrenata how do you feel about pointing applicants that aren't accepted toward the workshop materials?

**Pull request merged**